data-prepper-plugins/otel-metrics-raw-processor/build.gradle

```diff
@@ -0,0 +1,44 @@
+/*
```

Comment:
You should probably be using the following license header here and in all the source files.

```
/*
  * Copyright OpenSearch Contributors
  * SPDX-License-Identifier: Apache-2.0
  */
```

The longer form is only for files modified from other projects. Data Prepper source files mistakenly used these and now we've updated them. If you have any source files modified from projects outside of Data Prepper you can use the longer header. Otherwise, please update to the short header.